examples/testing/src/test/scala/DOMExistanceTest.scala

```diff
@@ -0,0 +1,36 @@
+import scala.scalajs.js.Dynamic.global
+
+import org.scalajs.jasminetest.JasmineTest
+
+object DOMExistanceTest extends JasmineTest {
```

Comment:
Exist**e**nce with an 'e'. Existance is a pretty rare spelling. (Same in the file name, and in the HTML files.)